build.gradle

```diff
@@ -165,6 +164,9 @@ subprojects {
     // Minus: some compilation errors (esp. from Checker) and stack traces refer to line numbers from the delomboked source code,
     //        which may differ a lot from the original source
     apply plugin: 'io.freefair.lombok'
+    lombok {
```

Comment:
Unrel, just made the version consistent (they happened to be consistent now anyway coz Lombok plugin 5.1.0 pulls in Lombok 1.18.12, the same that we used in the plugin-less mode)